modules/gapi/include/opencv2/gapi/own/assert.hpp

```diff
@@ -18,6 +18,7 @@
 #if !defined(GAPI_STANDALONE)
 #include <opencv2/core/base.hpp>
 #define GAPI_Assert CV_Assert
+#define GAPI_Error CV_Error
```

Comment:
These changes do not make sense in the framework of this PR since the error logging system should be added as a separate pull request. And also in this patch it is absolutely acceptable to use the opencv's error logging system, since it is under guard the `#if !defined(GAPI_STANDALONE)`.